### PR TITLE
Fix survey example

### PR DESCRIPTION
**Pull request by Chilliwiddit**
## Purpose
Create types.bal file
Make variables constant and remove redundant checks

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: examples/survey-feedback-analysis/main.bal [hunk 1]
```diff
@@ -20,32 +20,8 @@ import ballerinax/slack;
...
-};
-
-type RepliesResponse record {
-    boolean has_more;
-    MessagesItem[] messages;
-    boolean ok;
-    Response_metadata response_metadata;
-};
+final string channelName = "survey-coordination";
+final string surveyRequestMessage = "Reply to this survey message to give input on the company";
```
Comment (NipunaRanasinghe) [inline examples/survey-feedback-analysis/main.bal:24]:
```suggestion
const CHANNEL_NAME = "survey-coordination";
const SURVEY_REQUEST_MSG = "Reply to this survey message to give input on the company";
```

File: examples/survey-feedback-analysis/main.bal [hunk 2]
```diff
@@ -58,28 +34,28 @@ public function main() returns error? {
     // Create a new channel for the survey
     json|error createChannelResponse = check slack->/conversations\.create.post({name: channelName});
     if createChannelResponse is error {
-        log:printError("Error creating the survey conversation: ", createChannelResponse);
+        log:printError(createChannelResponse.message());
```
Comment (NipunaRanasinghe) [inline examples/survey-feedback-analysis/main.bal:37]: Please update all the similar places accordingly. 
```suggestion
        log:printError("Error creating the survey conversation: " + createChannelResponse.message());
```

**Review by NipunaRanasinghe — CHANGES_REQUESTED** [review]

File: examples/survey-feedback-analysis/main.bal (hunk 1 above)
Comment (NipunaRanasinghe) [reply examples/survey-feedback-analysis/main.bal]: @Chilliwiddit just applying the suggestion won't work :) you have to update the usages as well

Comment (Chilliwiddit) [reply examples/survey-feedback-analysis/main.bal]: Yes Sorry Nipuna it slipped my mind. I just fixed the above usage

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**